tests/2017-10-01/microsoft.insights.tests.json

```diff
@@ -0,0 +1,36 @@
+{
+  "tests": [
+    {
+      "name": "components_pricingPlans - basic test",
+      "definition": "https://schema.management.azure.com/schemas/2017-10-01/Microsoft.Insights.json#/resourceDefinitions/components_pricingPlans",
+      "json": {		  
+		  "type": "Microsoft.Insights/components/pricingPlans",
+		  "location": "us-west",
+		  "apiVersion": "2017-10-01",
+		  "dependsOn": [
+			"[resourceId('Microsoft.Insights/components', 'test-component1')]"
+		  ],
+		  "properties": {
+			"PricingType": "Basic",
+			"DataVolumeCap": "10.0",
+			"WarningThreshold": "80",
+			"ResetHour": "11"
+		  }
+		}
+    },
+    {
+      "name": "components - basic test",
+      "definition": "https://schema.management.azure.com/schemas/2015-05-01/Microsoft.Insights.json#/resourceDefinitions/components",
+      "json": {
+        "apiVersion": "2015-05-01",
```

Comment:
>2015-05-01 [](start = 23, length = 10)

do you need this test case? this is not the new api-version.

Reply:
Yes, the one above it dependent on this one.